demo/src/components/Pane.js

```diff
@@ -60,13 +60,30 @@ export class PaneRight extends React.Component {
       const { outputState } = this.props;
 
       return (
-        <ResultDisplay outputState={outputState}>
+        <ResultDisplay orientation="pane__right" outputState={outputState}>
```

Comment:
I find this naming confusing, "pane__right" doesn't feel like an "orientation" to me, it's a class name for a subcomponent.

I'm not 100% sure what I prefer, maybe something like `resultPane="right"` or `resultPane="bottom"`, and then up in line 45 you could do 

```
className={`pane__${resultPane} model__output ...`}
```